index.html

```diff
@@ -1,42 +1,262 @@
+                <button type="button" name="See-Project">See Project</button>
+            </div>
+            <div class="cart">
+                <h2>Profesional Art <span>Printing Data</span> </h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>Html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button" name="See-Project">See Project</button>
+            </div>
+        </div>
+        <!--carts for desktop-->
+        <div class="cartt2">
+            <div class="cart" id="c1">
+                
+                    <button type="button" name="See-Project">See Project</button>
+            </div>
+            <div class="cart" id="c2">
+                <h2>Data Dashboard Healthcare</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>Html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div class="cart" id="c3">
+                <h2>Website Protfolio </h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>Html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div class="cart" id="c4">
+                <h2>Profesional Art <span>Printing Data More</span> </h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>Html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div class="cart" id="c5">
+                <h2>Data Dashboard Healthcare</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>Html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div class="cart" id="c6">
+                <h2>Website Protfolio</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>Html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <!--the about me section-->
+    <footer id="about">
```

Comment:
Please use the `<section>` tag for the `About` section instead of a `footer`. 
The website footer is the section of content at the ***very bottom*** of a web page. It typically contains a copyright notice, link to a privacy policy, sitemap, logo, and contact information.

Reply:
Good job following the previous reviewer's suggestions.